app/src/main/res/values/dimens.xml

```diff
@@ -15,6 +15,8 @@
   <dimen name="divider_margin_top">8dp</dimen>
   <dimen name="divider_margin_bottom">8dp</dimen>
   <dimen name="bottom_white_space">160dp</dimen>
+  <dimen name="text_input_layout_corner_radius">4dp</dimen>
+  <dimen name="text_input_layout_storke_width">1dp</dimen>
```

Comment:
One more small change: Should be `stroke`.

Reply:
nice catch